### PR TITLE
Improve portfolio base on given comments and update week 3 progress

### PR DESCRIPTION
**Pull request by leeannachen**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -200,37 +228,39 @@ span.m-left{
...
+
+  margin: 10px;
 }
 
 .about-me-info p:nth-child(2),
 .about-me-info p:last-child,
 .contact-me p:nth-child(2),
 .contact-me p:last-child  {
-  margin-top:30px;
+ margin-top:30px;
```
Comment (aokeke) [inline portfolio/src/main/webapp/style.css:239]: I don't. think this change was necessary.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -17,16 +17,43 @@
...
+    quotes.add("Those who can imagine anything, can create the impossible. - Alan Turing");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String quote = quotes.get((int) (Math.random() * quotes.size()));
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Lee!</h1>");
```
Comment (aokeke) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Is this `<h1>Hello Lee!</h1>` Supposed to be part of the response? Couldn't you move it to the HTML code and just return the quote here?

**Review by aokeke — CHANGES_REQUESTED** [review]
Just a few very small things.

**Review by leeannachen — COMMENTED** [review]
change made based on review

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (leeannachen) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You are right, the html code was also returned as part of the quote.

**Review by leeannachen — COMMENTED** [review]
Review comments and changes

**Review by aokeke — APPROVED** [review]
LGTM

**Pull request merged**